src/app/csc/csc.component.html

```diff
@@ -52,19 +52,32 @@ <h4 class="loading">
           />Loading {{ title }} ScienceBase projects&hellip;
         </h4>
       </div>
-      <div
-        class="no_data"
-        *ngIf="filteredCscProjectsList.length === 0 && dataLoading === false"
-      >
+      <div class="no_data" *ngIf="!dataLoading && dataSource.data.length === 0">
```

Comment:
Also we kinda gotta update the code to account for this special case: "The user entered to text to filter by term, but there are zero matches."  Like, enter gibberish into the text field; the table is just empty but I bet a nice experience is to give a note like the one below.

Reply:
I have rebased the #204 into this PR, so I'm going to close #204 and this will be the PR with both the number of projects returned and the search bar.